fix(influxdb3): `api/v3/configure/database` requires format (#6714)
influxdata/docs-v2: content/shared/influxdb3-admin/databases/list.md

| iox::database       |
+---------------------+
| _internal           |
| home                |
| home_actions        |
| noaa                |
+---------------------+
```

{{% /tab-content %}}

{{% tab-content %}}

Include the following in your request:

- **Headers**:
  - `Authorization: Bearer` with your {{% token-link %}}
- **Query Parameters**:
  - `format=json`

```bash{placeholders="AUTH_TOKEN"}
curl --request GET "{{< influxdb/host >}}/api/v3/configure/database?format=json" \
  --header "Authorization: Bearer AUTH_TOKEN"
```

The response body contains a JSON array of database objects whose keys are `iox::database`:

```json
[
  {
    "iox::database": "home"
  },
  {
    "iox::database": "home_actions"
  },
  {
    "iox::database": "noaa"
  }
]
```
{{% /tab-content %}}

{{% tab-content %}}

Include the following in your request:

- **Headers**:
  - `Authorization: Bearer` with your {{% token-link %}}
- **Query Parameters**:
  - `format=parquet`
- An output destination for the Parquet file

```bash{placeholders="AUTH_TOKEN"}
curl "{{< influxdb/host >}}/api/v3/configure/database?format=parquet" \
  -o databases.parquet \
  --header "Authorization: Bearer AUTH_TOKEN"
```

> Parquet responses contain the same database list, but you must provide an output destination because the format is binary.

Author: ```suggestion
For Parquet responses, you must provide an output destination because the format is binary.
The response contains the databases list.
```

   - Creation date

> For more information, see [Manage databases with InfluxDB 3 Explorer](/influxdb3/explorer/manage-databases/).

Reviewer: The query subcommand actually _does_ support `--output` option in 3.8.

I opened a pr to support `--output` for the `show` subcommands too. That will be in 3.9. 

* see https://github.com/influxdata/influxdb_pro/pull/2120

Author: @copilot Revert and uncomment lines 145-158 to show the `query` example for parquet output, consistent with https://github.com/influxdata/docs-v2/pull/6714/files#r2683462350
